packages/browser/src/remote_settings.ts

```diff
@@ -103,7 +107,7 @@ export class RemoteSettings {
   _pollUrl(opt: IOptions): string {
     const url = new URL(this._data.configRoute(opt.remoteConfigHost));
 
-    for (const [key, value] of Object.entries(NOTIFIER_INFO)) {
+    for (const [key, value] of this._entries(NOTIFIER_INFO)) {
       url.searchParams.append(key, value);
```

Comment:
I think this line might need a polyfill as well. `URL.searchParams` is not supported by IE. https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams#browser_compatibility

Reply:
Thanks, that's a good idea.